TTHAnalysis/python/plotter/functions.cc

```diff
@@ -175,403 +185,59 @@ float DPhi_CMLep_Zboost(float l_pt, float l_eta, float l_phi, float l_M, float l
+float _puw2016_nTrueInt_G_upto279931[60] = {0.0030664708563046326, 0.01123794331581895, 0.016465056588825536, 0.024602832177926543, 0.026107749406095258, 0.019050503081821886, 0.02110161688725923, 0.017694605650846847, 0.016290999589588673, 0.03017245306638949, 0.062201175397462435, 0.21783085244166905, 0.5552753191757233, 0.7214955162768668, 0.7324485837208575, 0.7293440437992165, 0.7146681538266634, 0.7527382158346893, 0.7180480259007131, 0.7910622869835229, 0.7597422022206722, 0.7786794509943739, 0.8873764384591624, 1.0499884204081869, 1.1665585452761693, 1.482186697601494, 1.579761549660652, 1.9529819332323881, 2.053483793295554, 2.356853951860021, 2.5540748462771266, 2.7323308232944745, 3.76828118550158, 4.585241234971839, 5.680781421298886, 5.680781421298886, 5.680781421298886, 5.680781421298886, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995, 1.0225406558337995};
+float _puw2016_nTrueInt_EFG_upto279931[60] = {0.0012466094493921048, 0.013160935994534766, 0.01768708902983447, 0.021285749046576714, 0.02682365314703701, 0.020855099389744048, 0.020605061123741098, 0.016225941568981307, 0.014048132697170798, 0.02723810756281671, 0.08766748594327597, 0.2188586889136281, 0.3959226446275465, 0.53375637601322, 0.668298874408685, 0.7522202102811592, 0.7512257707056236, 0.8045384529297095, 0.790321346838139, 0.8694715549669896, 0.80985012101077, 0.8077764909536846, 0.9071025263393598, 1.0638089303658826, 1.17351814974311, 1.478754892106334, 1.5559650574410981, 1.8900984223481412, 1.9531327854179075, 2.2173732038373344, 2.402868150348858, 2.6038209490463284, 3.685234828496249, 4.657038942103952, 5.7091311002033045, 5.7091311002033045, 5.7091311002033045, 5.7091311002033045, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948};
+float puw2016_nTrueInt_EF(int nTrueInt) { if (nTrueInt<60) return _puw2016_nTrueInt_EF[nTrueInt]; else return 0; }
+float puw2016_nTrueInt_G_upto279931(int nTrueInt) { if (nTrueInt<60) return _puw2016_nTrueInt_G_upto279931[nTrueInt]; else return 0; }
+float puw2016_nTrueInt_EFG_upto279931(int nTrueInt) { if (nTrueInt<60) return _puw2016_nTrueInt_EFG_upto279931[nTrueInt]; else return 0; }
```

Comment:
@peruzzim: I'd move these to a different .cc file and put a comment explaining for which xsec they were computed.